src/openrct2-ui/audio/AudioChannel.cpp

```diff
@@ -118,42 +103,30 @@ namespace OpenRCT2::Audio
             return false;
         }
 
-        virtual int32_t GetLoop() const override
-        {
-            return _loop;
-        }
+        [[nodiscard]] int32_t GetLoop() const override { return _loop; }
 
-        virtual void SetLoop(int32_t value) override
+        void SetLoop(int32_t value) override
         {
             _loop = value;
         }
 
-        int32_t GetVolume() const override
-        {
-            return _volume;
-        }
+        [[nodiscard]] int32_t GetVolume() const override { return _volume; }
 
-        float GetVolumeL() const override
+            [[nodiscard]] float GetVolumeL() const override
         {
             return _volume_l;
         }
 
-        float GetVolumeR() const override
-        {
-            return _volume_r;
-        }
+        [[nodiscard]] float GetVolumeR() const override { return _volume_r; }
 
-        float GetOldVolumeL() const override
+            [[nodiscard]] float GetOldVolumeL() const override
```

Comment:
Is it just GitHub or are there extra spaces, clang-format doesn't complain.

Reply:
This must be a bug in an old clang-format since this ís the clang-format diff applied to my original.